portfolio/src/main/java/com/google/sps/servlets/TranslationServlet.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/translate")
+public class TranslationServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Gson gson = new Gson();
+    String[] comments;
+    String languageCode = request.getParameter("languageCode");
+    Translate translate = TranslateOptions.getDefaultInstance().getService();
+
+    comments = request.getParameterValues("comments");
```

Comment:
Could you get rid of comments declaration in line 19 and combine it with this assignment.